packages/core/src/Drawer/helpers/getDragToDismissAnimation.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * Handles header collapse/expand behavior on button click
+ */
+
+export const dragToDismissAnimation = (onDragEnd) => {
+  return {
+    drag: 'y',
+    dragConstraints: { top: 0, bottom: 0 },
+    dragElastic: { top: 0, bottom: 0.3 },
+    transition: { duration: 0.8, ease: [0.04, 0.62, 0.23, 0.98] },
+    onDragEnd: (event, info) => {
+      if (info.offset.y > 200) {
+        onDragEnd()
+      }
+    },
```

Comment:
Unit tests coming soon

Reply:
Storybook interaction tests would be ideal for this